jwst/cube_build/src/cube_match_internal.c

```diff
@@ -0,0 +1,484 @@
+/*
+The detector pixels are represented by a 'point could' on the sky. The IFU cube is
+represented by a 3-D regular grid. This module finds the point cloud members contained
+in a region centered on the center of the cube spaxel. The size of the spaxel is spatial
+coordinates is cdetl1 and cdelt2, while the wavelength size is zcdelt3.
+This module uses the e modified shephard weighting method to determine how to  weight each point clold member
```

Comment:
do you really mean "the *e* modified shepard weighting" or is the "e" extraneous? And another instance of "clold" that should be "cloud".